docs(source-from-drupal): Remove opinion text and reference Drupal module (#22219)
gatsbyjs/gatsby: docs/docs/sourcing-from-drupal.md

[“Decoupled Drupal”](https://www.acquia.com/drupal/decoupled-drupal) has become an increasingly popular approach to building enterprise-grade websites, and has the [full-throated support](https://dri.es/how-to-decouple-drupal-in-2018) of Drupal community leaders. Using Gatsby in a decoupled Drupal setup allows your team to access the powerful content modeling and access workflow capabilities of Drupal 8, as well as the powerful UI creation & performance toolset of Gatsby.

## Configuring Gatsby Live Preview

> The [Gatsby Drupal Module](https://www.drupal.org/project/gatsby) enhances the experience of your Drupal content editors for Gatsby sites using Drupal as a data source. This module provides Gatsby live preview capabilities using your Gatsby Cloud account or on your locally running Gatsby development server.

Reviewer: ```suggestion
The [Gatsby Drupal Module](https://www.drupal.org/project/gatsby) enhances the content editor experience when using Gatsby with Drupal as a data source. This module provides Gatsby live preview capabilities using your [Gatsby Cloud account](https://www.gatsbyjs.com/get-started) or on your [locally running Gatsby development server](/docs/running-a-gatsby-preview-server/).
```
